feat: New Component Float Button (#37520)
ant-design/ant-design: components/float-button/demo/group.md

---
order: 3
iframe: 360
title:

>   zh-CN: 多个悬浮按钮组合
>   en-US: Group of FloatButton

Reviewer: ```suggestion
  en-US: Group of FloatButton
```
```suggestion
  zh-CN: 浮动按钮组
  en-US: FloatButton Group
```

---
order: 4
iframe: 360
title:
  zh-CN: 浮动按钮组
  en-US: FloatButton Group
---

## zh-CN

> 按钮组合使用时，推荐使用 `<FloatButton.Group />`，你可以通过设置 shape 属性自定义悬浮按钮组的样式，悬浮按钮组的 shape 会覆盖内部的 shape 属性。

Reviewer: ```suggestion
按钮组合使用时，推荐使用 `<FloatButton.Group />`，并通过设置 `shape` 属性改变悬浮按钮组的形状。悬浮按钮组的 `shape` 会覆盖内部 FloatButton 的 `shape` 属性。
```

## en-US

> When multiple buttons are used together，you can use `<FloatButton.Group />`. You can customize the shape of the group by setting the `Shape` property. The shape of the group overrides the internal shape property.

Reviewer: ```suggestion
When multiple buttons are used together, `<FloatButton.Group />` is recommended. By setting `shape` of FloatButton.Group, you can change the shape of group. `shape` of FloatButton.Group will override `shape` of FloatButton inside.
```
